src/client.lisp

```diff
@@ -64,12 +64,26 @@
                      (rpc-protocol-error-object condition)))))
 
 
+(defun prepare-rpc-call-args (args)
```

Comment:
This function is all-around odd and I think it indicates a design flaw in rpcq.

Reply:
I'd love to have this discussion, but I'd also love to get this PR in and to involve others (e.g., @stevenheidel), so maybe this discussion should happen in a separate issue.